app/javascript/components/Projects/__tests__/ProjectsList.test.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  releasedProjectFixture,
+  unreleasedProjectFixture,
+} from "../../../fixtures/project"
+import { ProjectsList, ProjectsListRow } from "../ProjectsList"
+import { CircleBlackCheckIcon, Theme, XCircleIcon } from "@artsy/palette"
+import React from "react"
+import { mount } from "enzyme"
+
+describe("ProjectsList", () => {
+  let props
+  let component
```

Comment:
This feedback is late, so apologies on that, but I think it's idiomatic to call this `wrapper`:

https://enzymejs.github.io/enzyme/docs/api/

Or at least that's what the docs do so that's what I've done over on Volt, etc.

Reply:
We use `component` in a number of js projects including Force/Reaction/Positron- leaving for now.

Reply:
Huh, interesting!